.rubocop_todo.yml

```diff
@@ -0,0 +1,128 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2022-02-23 12:58:32 UTC using RuboCop version 1.10.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
```

Comment:
Should we at least apply the auto correctable fixes here?

Reply:
I've corrected all safe offenses. I'm not 100% sure if the unsafe ones couldn't cause some problems down the line?